CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## unreleased
```

Comment:
We've deprecated this changelog file in favour of using GitHub's release notes. Could you remove this entry?

Reply:
Okay never mind, looks like Zach merged this already. I can do this.

Reply:
Ohhh wait, I'm totally wrong, thought we did this here as well as `doctl`, but it's only for `doctl`. False alarm everyone 🤦‍♂